### ISSUE TITLE
[Feature Request] Add setting minimum segment time for Sponsorblock

### ISSUE THREAD

**Issue by kylagit** [yuliskov/SmartTube#1772]
It can be pretty annoying if Sponsorblock is constantly skipping/popping up, an option to set the minimum amount of seconds so that short segments are not skipped would make Sponsorblock a lot less annoying

Eg. If the minimum time is set to 5 seconds, a 6 second segment would be skipped, but a 4 second segment wouldn't

**This is a feature suggestion not an issue**

**Comment by albino1** [comment]
Related:

https://github.com/yuliskov/SmartTubeNext/issues/1662
https://github.com/yuliskov/SmartTubeNext/issues/443
https://github.com/yuliskov/SmartTubeNext/issues/385

**Comment by kylagit** [comment]
The first one is exactly what i'm asking about! Please add this in the
next version.

**Comment by arantius** [comment]
I was about to report the same request!  So instead, I'll add my detail here.

I'm seeing this e.g. [here](https://sponsor.ajay.app/api/skipSegments?videoID=5Ka7dAPMWq8):

	{
		"category":"sponsor",
		"actionType":"skip",
		"segment":[
			129.311,
			130.842
		],
		"UUID":"35e84a55d57614e4d4b05cdaf3849d1c4c6ec64812ed945e786a48c4c54693697",
		"videoDuration":1130.361,
		"locked":0,
		"votes":0,
		"description":""
	},

This is a ~1.5 (just barely more) second segment, which I'd prefer to ignore (not skip).

I also think https://github.com/yuliskov/SmartTube/issues/1946 is related -- I could implement filtering and/or combining (like https://github.com/yuliskov/SmartTube/issues/2308 also mentions) of segments in my own server.

**Comment by DarkwaterV3** [comment]
Chiming in to say this would be a very welcome feature for me. Extremely short skips and skip prompts are more annoying than whatever could be skipped over.

Some people really go overboard with their SponsorBlock submissions and I realize there's no stopping that, really. This option would turn it into a non-issue.

**Comment by fluffy-critter** [comment]
The original SponsorBlock browser extension supports this and I'd really love it if SmartTube did too. I'm seeing so many videos now where people are just zealously tagging every single two-second mention of the video sponsor, even if it's relevant to the content, and the constant skipping/muting/loss-of-three-words-in-a-sentence is way more disruptive to the video than the sponsorship itself.

**Comment by albino1** [comment]
Yes, exactly.  The tagging is out of control.  I mentioned this in [another issue](https://github.com/yuliskov/SmartTube/issues/1662), but besides people tagging every time someone even briefly says the name of a sponsor, you have people on things like travel videos tagging the names of the places they're visiting causing them to get skipped so you don't even know where they are.  Or tagging movie title cards during movie reviews so you don't even know what movie they're talking about until they say the name.

All of that would be solved with a minimum segment time that doesn't auto-skip and doesn't show the popup.

**Comment by fluffy-critter** [comment]
Ah yeah looks like this issue is a duplicate of #1662, it would be great if these could be merged

**Comment by Karasuni** [comment]
Some of the closed issues did not match this topic exactly (#385)

**Comment by ACR-Jeff** [comment]
> Some of the closed issues did not match this topic exactly ([#385](https://github.com/yuliskov/SmartTube/issues/385))

You're Correct, I stand corrected and fixed it, Thank you, Human error LOL
